src/test/migrations/steps/20230511173240-step-pan-european-degradation-rowspan-fix.ts

```diff
@@ -0,0 +1,25 @@
+import { Row, Table } from '@meta/assessment'
+
+import { AssessmentController } from '@server/controller/assessment'
+import { BaseProtocol, Schemas } from '@server/db'
+
+export default async (client: BaseProtocol) => {
+  const { assessment, cycle } = await AssessmentController.getOneWithCycle(
+    { assessmentName: 'panEuropean', cycleName: '2025' },
+    client
+  )
+
+  const schemaName = Schemas.getName(assessment)
+
+  const table = await client.one<Table>(`select * from ${schemaName}.table where props->>'name' = $1;`, 'table_2_5')
```

Comment:
oh wait 😅 pan European is still using panEuropeanSpecs, there we still run the old way. this won't work in that case . didn't realize it was a metadata issue, I thought it was a css problem